BDB-Atlas-Configs

SkyPhoenix999 opened the pull request:
Accidently Deleted the Old Branch and PR at the same time, this restores it. _This time to the Proper Branch_

Renamed the Folder and Files for the Old Atlas Parts since the Atlas Update for BDB is Coming.

There are no changes to these configs from the last version of the former accidently deleted Atlas PR other than renaming the folder and files.

SkyPhoenix999 committed:
BDB-Atlas-Configs

RO Configs for the Atlas Parts in BDB

SkyPhoenix999 committed:
Changed Temp Values

SkyPhoenix999 committed:
Fat Trim

SkyPhoenix999 committed:
Updated Temp Values

SkyPhoenix999 committed:
Updated Temp Values

Capkirk123 committed:
Improve Atlas configs

SkyPhoenix999 committed:
Delete GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Atlas/Waterfall_Support_Atlas.cfg

Deleting so I can move it

SkyPhoenix999 committed:
Moved Waterfall Config

Capkirk123 committed:
Correct mass since we fixed it in BDB

SkyPhoenix999 committed:
RO_Materials Support and Fixed Atlas Roll Control Part

RCS for the ARCM didn't work. Fixed it

SkyPhoenix999 committed:
RO_Materials Support for CELV

SkyPhoenix999 committed:
RO_Materials for SLV-3X

SkyPhoenix999 committed:
B9PartSwitch retitling for Long Adapter Tank

SkyPhoenix999 committed:
Update RO_Atlas.cfg

Updated Fuel Volume and Mass Values

SkyPhoenix999 committed:
Update RO_Atlas_CELV.cfg

Updated Fuel Volume and Mass Values

SkyPhoenix999 committed:
Update RO_Atlas_SLV3X.cfg

Updated Fuel Volume and Mass Values

SkyPhoenix999 committed:
Update and rename RO_Atlas.cfg to RO_Atlas_OLD.cfg

Renaming files for the Atlas Update

SkyPhoenix999 committed:
Update and rename RO_Atlas_CELV.cfg to RO_Atlas_CELV_OLD.cfg

Renaming files for the Atlas Update

SkyPhoenix999 committed:
Update and rename RO_Atlas_Engines.cfg to RO_Atlas_Engines_OLD.cfg

Renaming files for the Atlas Update

SkyPhoenix999 committed:
Update and rename RO_Atlas_SLV3X.cfg to RO_Atlas_SLV3X_OLD.cfg

SkyPhoenix999 committed:
Rename RO_Waterfall_Atlas.cfg to RO_Waterfall_Atlas_OLD.cfg

SkyPhoenix999 committed:
Delete GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Atlas/TU_Support_Atlas.cfg

This thing has never worked properly, just get rid of it, someone else will make TU work properly

SkyPhoenix999 committed:
Update and rename RO_Atlas_CELV_OLD.cfg to RO_Atlas_CELV_OLD.cfg

New Folder Name

SkyPhoenix999 committed:
Update and rename RO_Atlas_Engines_OLD.cfg to RO_Atlas_Engines_OLD.cfg

Moving Files

SkyPhoenix999 committed:
Update and rename RO_Atlas_OLD.cfg to RO_Atlas_OLD.cfg

Moving Files

SkyPhoenix999 committed:
Update and rename RO_Atlas_SLV3X_OLD.cfg to RO_Atlas_SLV3X_OLD.cfg

Capkirk123 commented:
I already went over this and not much has changed since then so it's probably fine

SkyPhoenix999 committed:
Update for New Atlas Parts

Configs for New Atlas Parts

Old Parts moved to subfolder

SkyPhoenix999 commented:
Updated RO Configs for the new Atlas Parts

Old Parts were moved to a Subfolder

SkyPhoenix999 committed:
Update to add OV1 Parts

- Added OV1 Fairing and Decoupler
- Atlas H Decoupler
- Removed Extras Config that wasn't meant to be included
